backend.native/tests/build.gradle

```diff
@@ -2879,6 +2879,20 @@ task inline_defaultArgs_linkTest(type: LinkKonanTest) {
     lib = "codegen/inline/defaultArgs_linkTest_lib.kt"
 }
 
+def generateWithSpaceDefFile() {
```

Comment:
This generated file is treated by Git as a new one untracked file. So I think it's better to generate it in the `build` directory which is ignored by Git.